turbopack/crates/turbopack-node/src/transforms/webpack.rs

```diff
@@ -556,16 +569,23 @@ impl EvaluateContext for WebpackLoaderContext {
                 .collect();
 
             EvaluateErrorLoggingIssue {
-                file_path: self.context_ident_for_issue.path(),
+                file_path: self.context_ident_for_issue.path().to_resolved().await?,
                 logging: logs,
                 severity: if has_errors {
                     IssueSeverity::Error.cell()
                 } else {
-                    IssueSeverity::Warning.cell()
-                },
+                    IssueSeverity::Warning.into()
```

Comment:
We're actually slowly trying to get rid of these `.into()` calls for cell construction, as they're less obvious/explicit than `.cell()` or `.resolved_cell()` and they're not compatible with non-public cell constructors. Tobias wants to get rid of the `From` derivation.

I've been cleaning them up as I see them. Either way, this should use `.resolved_cell()` instead of the `.to_resolved()` below.